### ISSUE TITLE
Loading PVAR in pgenlibr

### ISSUE THREAD

**Issue by GabrielHoffman** [chrchang/plink-ng#286]
I'm working on an R package that reads PGEN files and performs statistical analysis in C++ via Rcpp (https://github.com/chrchang/plink-ng/issues/283).  `pgenlibr` provides almost everything a user needs.

The R function `pgenlibr::NewPvar()` to load a `.pvar` file calls `LoadMinimalPvar()` in C++:

https://github.com/chrchang/plink-ng/blob/6287f6a083650fcad2709ba1689beda39680e367/2.0/pvar_ffi_support.cc#L64

It loads `ID`, `REF` and `ALT`, but not `CHR` or `POS`.

Do you have code to include chromosome and position information, so a user can intersect the variants with a BED file of genome intervals?

I'd do this myself, but that function is very hard to follow, and I'd write a new version but I'm concerned that there are some `.pvar` values it wouldn't parse correctly.

Best,
Gabriel

**Comment by chrchang** [comment]
This wasn't present in the first released version of pgenlibr since --set-all-var-ids provided a simple way to work around the limitation, but okay, now it's embarrassing.  I will add this sometime in the next two weeks.

**Comment by chrchang** [comment]
This capability is in the current pgenlibr (0.4.0) and Python Pgenlib (0.92.0).
